packages/amplify-category-api/src/provider-utils/awscloudformation/container-artifacts-graphql.js

```diff
@@ -0,0 +1,132 @@
+export const containerFiles = {
+    Dockerfile: `FROM node:alpine
+
+WORKDIR /usr/src/app
+COPY package*.json ./
+RUN npm install
+COPY . .
+
+EXPOSE 3000
+
+CMD ["node", "index.js"]
+    `,
+    'index.js': `'use strict';
+
+const PORT = 3000;
```

Comment:
maybe use an uncommon port here? Just an issue when running docker locally - for example projects like CRA also spin up the server on port 3000 and also amplify mock I believe  uses this port.

Reply:
Sure that's fair but just to be clear this is more the port for running in Fargate and less about the client operation. We can change to 3001 or something.